docs/appendix/contributing.md

```diff
@@ -52,6 +52,131 @@ When in doubt, ask during review.
 All documentation is in the [docs folder](https://github.com/viamrobotics/docs/tree/main/docs).
 For information about Hugo and how to develop locally, see the [README](https://github.com/viamrobotics/docs/blob/main/README.md).
 
+## Content Types
+
+The docs use the [Diátaxis Framework](https://diataxis.fr/) as the basis of the content structure with the following four content types:
+
+- **Explanation (conceptual)**: An understanding-oriented piece of content.
+  This content provides background knowledge on a topic and tends to be linked to from how-to guides and tutorials.
+  For example the [Robot Development Kit page](https://docs.viam.com/internals/rdk/) or the [Registry page](https://docs.viam.com/registry/).
+  It’s useful to have a real or imagined "Why?" question to serve as a prompt.
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Concept
+
+  Introductory paragraph.
+  Possibly containing further subsections or links to relevant conceptual information.
+
+  ## Subconcept
+
+  Paragraphs containing explanation. Add imagery as needed.
+
+  (possibly more Subconcept sections)
+
+  ### Do x with subconcept (optional)
+
+  Information for what to do with the concept (possibly linking to how-tos or containing how-tos).
+
+  ## Next Steps
+
+  Links to related content.
+  ```
+
+  {{% /expand %}}
+
+- **How-to Guide (procedural)**: A task-oriented piece of content that directs a reader to perform actions step by step to complete a task, like a recipe.
+Generally starts with a description of the task and things to consider and then provides a set of numbered steps to follow.
+For example, the [Installation page](https://docs.viam.com/get-started/installation/) or the [Find module page](https://docs.viam.com/registry/configure/).
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Do this task
+
+  Description of task. Possibly containing further subsections.
+
+  ## Do x
+
+  1. Ordered list of actions to perform. Written in imperative form. Add imagery as needed.
+
+  (possibly more Do X sections)
+
+  ## Next steps
+
+  Links to related content.
+  ```
+
+  {{% /expand %}}
+
+- **Tutorial**: A learning-oriented piece of content that functions as a lesson for the reader.
+A tutorial helps readers to learn skills by doing something meaningful and attainable.
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Do X with Y
+
+  Outline the why.
+  Tell the story of the machine.
+
+  ## Requirements
+
+  What does the reader need to already know.
+  What will you be using (hardware/software).
+
+  ## Build X
+
+  Build steps
+
+  ## Configure your X
+
+  Configuration steps
+
+  ## Program your X
+
+  Code and directions.
+
+  ## Next steps
+
+  Link to other tutorials with cards or text.
+
+  {{< cards >}}
+  {{% card link="/tutorials/get-started/blink-an-led" %}}
+  {{< /cards >}}
```

Comment:
Think we need to escape these curly brackets to get this to render as-is within code fences. I think the following should work here:

```suggestion
  {{</* cards */>}}
  {{%/* card link="/tutorials/get-started/blink-an-led" */%}}
  {{</* /cards */>}}
```